### PR TITLE
feat: add social links container component

### PR DESCRIPTION
**Pull request by edraven-dev**

### CODE REVIEW

File: src/components/social-links/SocialLinks.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
+@import "../../styles/colors";
+
+#social-links {
```
Comment (Ciszak-Krzysztof) [inline src/components/social-links/SocialLinks.scss:3]: we are usually using classes for styling, not ids. With Ids you have to remember that it has higher specificity ( id, class, element )

**Review by Ciszak-Krzysztof — APPROVED** [review]

File: src/components/social-links/SocialLinks.scss (hunk 1 above)
Comment (rafalciecwierzmoodup) [reply src/components/social-links/SocialLinks.scss:3]: Try to rewrite it for classes (yeah - tags to ;) )

Comment (edraven-dev) [reply src/components/social-links/SocialLinks.scss:3]: that was my main idea - to encapsulate the styles within the component element only

Comment (edraven-dev) [reply src/components/social-links/SocialLinks.scss:3]: in other words - I wanted to achieve something like styled-components with pure SASS

**Pull request merged**